atlp-rwanda/rca-barefoot-team3-fe
# #184637959 Users should be able to search

### What does this PR do
This PR fetch bookings and display them in table and allow user to search bookings either using tabs above or search input 
### Description of Task to be completed?
1.fetch bookings and display them in table
2.if user select status filter bookings by their status
3.if user  search using input filter by comparing every property of bookings that contains the searched value
### How should this be manually tested?
1. first login  
2. got /admin to the dashboard
3. click bookings and make sure you have bookings in database 

### Any background context you want to provide?
No

### What are the relevant pivotal tracker/Trello stories?
 #184637959 -Users should be able to search 

Screenshots (if appropriate)


![image](https://user-images.githubusercontent.com/76569213/235649768-cc9addd7-0618-41f5-99c1-7ea8080fa29c.png)
